client/.stylelintrc.js

```diff
@@ -104,7 +104,9 @@ module.exports = {
 
```

Comment:
__Confession__
I do not remember what these changes do, but I know they let me write CSS—used in this PR—in a syntax I believe would help others read the code more easily. _I did not change these values because "I like to write this way for me"._